viewer/viewer.js

```diff
@@ -9721,7 +9721,7 @@ function processCronQueries () {
 // Main
 // ----------------------------------------------------------------------------
 function main () {
-  if (!fs.existsSync('./vueapp/dist/index.html')) {
+  if (!fs.existsSync('./vueapp/dist/index.html') && app.settings.env !== 'development') {
```

Comment:
How does the env account to whether index.html exists?

Reply:
If the app is run in development mode, webpack blows away the vueapp/dist directory, so the index file is not there and this error always happens. If it's run in production, the build is run before running the app and the index file is already there.